website/source/docs/agent/checks.html.md

```diff
@@ -366,3 +366,21 @@ key in your configuration file.
   ]
 }
 ```
+
+## Success/Failures before passing/critical
+
+A check may be set to become passing/critical only if a specified number of consecutive
+checks return passing/critical. Status will stay identical as before until
+the threshold is reached.
+This feature is available for HTTP, TCP, gRPC, Docker & Monitor checks.
+By default, will be set to 0 for both passing & critical - a single check is needed to update status.
```

Comment:
```suggestion
By default, both passing and critical thresholds will be set to 0 so the check status will always reflect the last check result.
```

Reply:
@PHBourquin are you able to commit this tweak? If not I can make it but I can't commit to your fork so it makes it a bit more cumbersome.

Reply:
On it